WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/EditAttributesViewController.swift

```diff
@@ -15,7 +15,11 @@ final class EditAttributesViewController: UIViewController {
 
     /// Assign this closure to be notified after a variation is created.
     ///
-    var onVariationCreation: ((ProductVariation) -> Void)?
+    var onVariationCreation: ((Product) -> Void)?
```

Comment:
nit: maybe we can also change the variable name passed to this closure? right now it's still called `variation` 🙂

Reply:
I prefer to leave it as it is because the closure is only invoked when a variation is created. I changed the associated variable because we don't really need the variation(as it's only an `ID` inside `Product`).